Formula/jumanpp.rb

```diff
@@ -0,0 +1,24 @@
+class Jumanpp < Formula
+  desc "Japanese Morphological Analyzer based on RNNLM"
+  homepage "http://nlp.ist.i.kyoto-u.ac.jp/EN/index.php?JUMAN%2B%2B"
+  url "http://lotus.kuee.kyoto-u.ac.jp/nl-resource/jumanpp/jumanpp-1.01.tar.xz"
+  sha256 "0d587416a3eb7123638f9c1e30a649b72dfb483448839168dcb48be572c5919a"
+
+  depends_on "boost-build" => :build
+  depends_on "gperftools" => :build
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    # Currently there is a bug in Makefile.
+    # `echo "" | src/jumanpp -D juman-resources` is too slow so that make process exits with error code.
+    # Preventing this bug, disable parallel build
+    system "make", "-j1"
```

Comment:
According to the Cook-book, it is required patches to be **submit**, not to be **merged**.

> If not, file a bug report and/or submit your patch for inclusion. We may sometimes still accept your patch before it was submitted upstream but by getting the ball rolling on fixing the upstream issue you reduce the length of time we have to carry the patch around.

And as I wrote, I already [submitted the patch to author on twitter](https://twitter.com/chezou/status/786753665671442433), or is there any rule that all softwares to be proposed to homebrew should have public issues? Unfortunately, they don't have any public repo at present.